tests/plugins/test_tpu_spawn.py

```diff
@@ -114,7 +114,6 @@ def test_tensorboard_ddp_spawn_cleanup(use_list, tmpdir):
     assert tensorboard_logger._experiment is not None
     logger = [tensorboard_logger] if use_list else tensorboard_logger
     trainer = Trainer(strategy="ddp_spawn", accelerator="tpu", devices="auto", logger=logger)
-    trainer.training_type_plugin._clean_logger(trainer)
```

Comment:
Wait do you know why this did not fail? Is it because it doesn't run at all?

Reply:
there is a file test_tpu_cases.jsonnet and it does not include the file these tests reside in. 
So yeah, RIP.

Reply:
I guess we should replicate the test search and filtering we do for `standalone=True` but for `tpu=True`

Reply:
doing that in #11098 👍